README_INTERNAL.md

```diff
@@ -1,9 +1,6 @@
 ## Releasing
 
-1. Update CHANGELOG.md
 1. Update version in package.json
 1. Open Pull Request, merge after approved
 1. Tag master: example: `git tag v2.6.0` and `git push --tags origin`
```

Comment:
Should we be suggesting doing this directly or using `npm version` to both bump package.json and create the tag?

Reply:
good shout, I like that better!